dbt/sagerx/models/marts/products/products_to_inactive_ingredients.sql

```diff
@@ -4,4 +4,71 @@ with products_to_inactive_ingredients as (
     select * from {{ ref('int_mthspl_products_to_inactive_ingredients') }}
 )
 
-select * from products_to_inactive_ingredients
+, unii_codes as (
+    select * from {{ ref('stg_fda_unii__unii_codes') }}
+)
+
+, usp_preservatives as (
+    select * from {{ ref('usp_preservatives') }}
+)
+
+select
+    ndc9
+    , ndc
+    , unii_codes.unii as fda_unii_code
+    , unii_codes.display_name as fda_unii_display_name
+    , unii_codes.pubchem as pubchem_id
+    , max(case
+        when preservative.cas_rn is not null
+            then 1
```

Comment:
Doing 1 or NULL here b/c I'm not 100% confident that it's TRUE or FALSE... really just TRUE or NULL.